panda:

  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController

    publish_rate: 50

  # Position Controllers ---------------------------------------
  panda_joint1_position_controller:
    type: position_controllers/JointPositionController
    joint: panda_joint1
    pid: {p: 450., i: 0.0, d: 20.}
  panda_joint2_position_controller:
    type: position_controllers/JointPositionController
    joint: panda_joint2
    pid: {p: 450.0, i: 0.0, d: 20.}
  panda_joint3_position_controller:
    type: position_controllers/JointPositionController
    joint: panda_joint3
    pid: {p: 450.0, i: 0.0, d: 20.0}
  panda_joint4_position_controller:
    type: position_controllers/JointPositionController
    joint: panda_joint4
    pid: {p: 450.0, i: 0, d: 20.0}
  panda_joint5_position_controller:
    type: position_controllers/JointPositionController
    joint: panda_joint5
    pid: {p: 80.0, i: 0.0, d: 1.5.0}
  panda_joint6_position_controller:
    type: position_controllers/JointPositionController
    joint: panda_joint6
    pid: {p: 20.0, i: 0.0, d: 1.0}
  panda_joint7_position_controller:
    type: effort_controllers/JointPositionController
    joint: panda_joint7
    pid: {p: 1.0, i: 0.0, d: 0.05.0}

/gazebo_ros_control:
  pid_gains:
    panda_joint1: {p: 450.0, i: 0.0, d: 20.0}
    panda_joint2: {p: 450.0, i: 0.0, d: 20.0}
    panda_joint3: {p: 450.0, i: 0.0, d: 20.0}
    panda_joint4: {p: 450.0, i: 0.0, d: 20.0}
    panda_joint5: {p: 80.0, i: 0.0, d: 1.5.0}
    panda_joint6: {p: 20.0, i: 0.0, d: 1.0}
    panda_joint7: {p: 1.0, i: 0.0, d: 0.05}
